Extension/src/LanguageServer/configurations.ts

```diff
@@ -1601,7 +1601,7 @@ export class CppProperties {
             let envText: string = "";
             const envStart: number = curText.search(/\"env\"\s*:\s*\{/);
             const envEnd: number = envStart === -1 ? -1 : curText.indexOf("},", envStart);
-            envText = curText.substr(envStart, envEnd);
+            envText = curText.substr(envStart, envEnd); // TODO: Replace with substring.
```

Comment:
It looks like this code is checking if `env` exists. If it does, then get the offset for `env`. If not, then there's no need to do a substring on curText to get envText, so the substring statement can be placed in a conditional block.